src/Model/Invoices/IssueData.php

```diff
@@ -2,6 +2,7 @@
 
 namespace App\Model\Invoices;
 
+use App\Entity\Epic;
```

Comment:
I don't think we should mix Entity with Model.
The collection $epics could just be an array of strings instead. Then you can map to Epic entity in DataSynchronizationService.